src/ui/components/Overlay/styles.scss

```diff
@@ -13,6 +13,12 @@
   position: fixed;
   top: 0;
   width: 100%;
+  z-index: 2;
```

Comment:
That looks a bit weird because the z-index for `.Overlay-contents` is at `1001`

Reply:
yeh but that's a different level in the DOM hierarchy so it's different..